### PR TITLE
feat: Block liquidity and show warning if user has a position on v1

### PR DESCRIPTION
**Pull request by Tulun**
Screen Shot 2022-04-20 at 11 46 26 AM

REACT_APP_MIGRATION_POOL_V2_WARNING must be set to "true" in  .env file.

### CODE REVIEW

**Review by Gamaranto — APPROVED** [review]

File: src/components/PoolSelection/PoolSelection.tsx [hunk 1]
```diff
@@ -54,6 +57,20 @@ const PoolSelection: FC<Props> = ({ token, setToken }) => {
   return (
     <AnimatePresence>
       <Wrapper>
+        {migrationPoolV2Warning && position.gt(0) ? (
+          <MigrationWarning>
+            <div>
+              You still have liquidity on v1 and some more text plus link to{" "}
```
Comment (daywiss) [inline src/components/PoolSelection/PoolSelection.tsx:63]: what is this saying?

**Review by daywiss — COMMENTED** [review]
looks good just somethign wrong with the copy

**Review by amateima — APPROVED** [review]

File: src/components/PoolSelection/PoolSelection.tsx (hunk 1 above)
Comment (Tommy1231232) [reply src/components/PoolSelection/PoolSelection.tsx:63]: @Tulun this is the same text as the other PR: "If you have not migrated liquidity from Across v1 to Across v2, please follow these instructions " We will link Lana's post here once it is done

Comment (Tulun) [reply src/components/PoolSelection/PoolSelection.tsx:63]: With what content @Tommy1231232  for the orange banner? or does this exist here.

There is a real mixing in Figma what's in V1 vs V2.

Comment (Tommy1231232) [reply src/components/PoolSelection/PoolSelection.tsx]: Lana will have a medium post we will link in the orange banner. It's not ready yet to my knowledge. Yes sorry this is for v1. 

1. Users will migrate liquidity from v1 to V2 (v2.across.to). 
2. when enough liquidity is migrated to v2 we will points v2.across.to to across.to
3. We show the orange banner for users to go to v1.across.to to migrate their liquidity if they haven't done so yet

Does this make sense?

**Pull request merged**